ironfish/src/network/version.ts

```diff
@@ -3,4 +3,4 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 export const VERSION_PROTOCOL = 18
-export const VERSION_PROTOCOL_MIN = 15
+export const VERSION_PROTOCOL_MIN = 17
```

Comment:
We bump this 2 because we did 2 versions of network protocol that are not incompatible?

Reply:
Looks like we bumped it once (15 -> 16) when we replaced the [gossip nonce filter with a transaction hash filter](https://github.com/iron-fish/ironfish/pull/1865) and bumped it again for [starting to gossip transactions based on hash](https://github.com/iron-fish/ironfish/pull/1803)

Reply:
So we did two bumps for the transaction changes. We may have been thinking we needed to bump the min version at the same time we made the change to start gossiping transactions by hash but we didn't end up doing that